src/pages/tabs/tabs.ts

```diff
@@ -11,8 +10,7 @@ export class TabsPage {
   // this tells the tabs component which Pages
   // should be each tab's root Page
   tab1Root: any = HomePage;
-  tab2Root: any = AboutPage;
-  tab3Root: any = ContactPage;
+  tab2Root: any = InventoryPage;
```

Comment:
It is true that it would increase clarity, but if we ever decide to change the name of the tabs, it would require extra work to change the name of the roots and could introduce potential problems. Might not be worth it.